deployment/README.md

```diff
@@ -123,3 +123,113 @@ The deployment depends on several credentials which are maintained using the [AW
 
 The deployment scripts will obtain this json document for the respective environment and expose them to terraform through several
 environment variables.
+
+
+## Maintenance
+
+The sections below describe actions relevant for advanced usage or maintenance of the deployment process.
+
+### Adding Users
+Adding new users that are able to perform deployments requires 2 individual steps:
+
+1. Creating an AWS account for the new user
+2. Adding the user's ssh key
+
+The AWS login is required in order to provision infrastructure using Terraform. The ssh key has to be added in order to enable users to perform deployments with morph through the ssh jump host.
+
+#### AWS
+The new user has to be added to the appropriate AWS organization. Please talk to a plutus/marlowe team member and request access. New
+users with appropriate permissions have to be added manually through the aws console.
+
+#### Terraform
+In order to perform nix deployments a ssh-key has to be configured in [deployment/terraform/locals.tf](https://github.com/input-output-hk/plutus/blob/master/deployment/terraform/locals.tf):
+
+**1.** **Create a new ssh keypair**:
+
+```
+$ ssh-keygen -t ed25519
+```
+
+**2.** **Add the user/key to the `ssh_keys` map in** [deployment/terraform/locals.tf](https://github.com/input-output-hk/plutus/blob/master/deployment/terraform/locals.tf)
+
+```
+
+ssh_keys = {
+    username = "ssh-ed25519 AAAAC...f3JfmL3A2 usernamer@host
+}
+```
+
+**3**. **Add the new user to environments that they should be able to deploy to**
+
+In order to allow the user (ssh key) to deploy to the `testing` environment [deployment/terraform/locals.tf](https://github.com/input-output-hk/plutus/blob/master/deployment/terraform/locals.tf) needs to be edited as shown below:
+
+```
+  bastion_ssh_keys_ks = {
+    testing = ["username"]
+    ...
+  }
+
+  root_ssh_keys_ks = {
+    testing = ["username"]
+    ..
+  }
+```
+
+### Adding Environments
+
+Deployments can be performed to different environments. Each environment is a full aws setup with multiple ec2 instances and networking, deployed to different `iohkdev.io` subdomains:
+
+- The `alpha` environment is deployed to `alpha.iohkdev.io`
+- `testing` is deployed to `testing.iohkdev.io`
+
+Terraform uses different workspaces for each environment which are also separated in the shared state which is stored in a S3 bucket. When entering a nix-shell the respective terraform workbench is chosen automatically.
+
+In order to add a new environment the following steps need to be followed:
+
+**1. Add the environment to [deployment/envs.nix](https://github.com/input-output-hk/plutus/blob/master/deployment/envs.nix)**
+
+In order to add an environment `username` it needs to be added to the attribute set in `deployment/envs.nix` as follows:
```

Comment:
A tiny thing, but maybe we could just have "environment" as the placeholder name for the environment. It needn't be a username (and `alpha` and `testing` aren't).